### PR TITLE
address editorial items from AD review

### PR DESCRIPTION
**Pull request by carl-wallace**

### CODE REVIEW

**Review by laurencelundblade — CHANGES_REQUESTED** [review]

File: draft-ietf-rats-eat.md [hunk 1]
```diff
@@ -222,28 +228,27 @@ Here are some examples:
...
 Correspondingly, the EAT format allows claims to be organized using mechanisms like submodules and nested EATs (see {{submods}}).
 The entity to which a claim applies is the submodule in which it appears, or to the top-level entity if it doesn't appear in a submodule.
 
-An entity also corresponds to a "system component", as defined in the Internet Security Glossary {{RFC4949}}.
-That glossary also defines "entity" and "system entity" as something that may be a person or organization as well as a system component.
-In the EAT context, "entity" never refers to a person or organization. The hardware and software that implement a server or service used by a web site may be an entity, but the organization that runs the web site is not an entity nor is the web site itself. An entity is an implementation in hardware, software or both.
+An entity also corresponds to a "system component", as defined in the Internet Security Glossary {{RFC4949}}, except that in the EAT context an "entity" never refers to a person or organization.
```
Comment (laurencelundblade) [inline draft-ietf-rats-eat.md:251]: https://github.com/ietf-rats-wg/eat/pull/387

File: draft-ietf-rats-eat.md [hunk 2]
```diff
@@ -129,14 +130,19 @@ normative:
 
   CycloneDX:
      title: CycloneDX
-     target: https://cyclonedx.org/specification/overview/
+     target: https://cyclonedx.org/docs/1.4/json/
```
[the remark this answers is not in the document]
Comment (carl-wallace) [reply draft-ietf-rats-eat.md:133]: No change given this is going away in #386 anyway.

File: draft-ietf-rats-eat.md [hunk 3]
```diff
@@ -1597,7 +1604,7 @@ The "bootseed" claim is effectively a stable entity identifier within a given bo
 
 ## Replay Protection and Privacy {#replayprivacyconsiderations}
 
-EAT defines the nonce claim for replay protection and token freshness.
+EAT defines the eat_nonce claim for replay protection and token freshness.
```
[the remark this answers is not in the document]
Comment (carl-wallace) [reply draft-ietf-rats-eat.md:1607]: Changed to "EAT nonce".

File: draft-ietf-rats-eat.md [hunk 4]
```diff
@@ -269,14 +274,14 @@ While EAT is based on and compatible with CWT and JWT, it can also be described
...
-* Claims are defined in CDDL and serialized using CBOR or JSON
+* Claims defined in CDDL and serialized using CBOR or JSON
 * Security envelopes based on COSE and JOSE
 * Nesting of claims sets and tokens to represent complex and compound devices
 * A profile mechanism for specifying and identifying specific token formats for specific use cases
 
 EAT uses the name/value pairs the same as CWT and JWT to identify individual claims.
 {{theclaims}} defines common attestation-oriented claims that are added to the CWT and JWT IANA registries.
-As with CWT and JWT, no claims are mandatory and claims not recognized should be ignored.
+The set of claims that a EAT must contain to be considered valid is context dependent and is outside the scope of this specification. Specific applications of EATs will require implementations to    understand and process some claims in particular ways.  However, in the absence of such requirements, all claims that are not understood by implementations MUST be ignored.
```
[the remark this answers is not in the document]
Comment (carl-wallace) [reply draft-ietf-rats-eat.md:284]: Reverted per your request.

File: draft-ietf-rats-eat.md (hunk 1 above)
Comment (carl-wallace) [reply draft-ietf-rats-eat.md:251]: I am assuming you are saying "leave it as-is and #387 will apply changes I prefer".

**Pull request merged**